### DISCUSSION TITLE
Voting on appropriate name for reference to compute resources identifier where applications are deployed

### DISCUSSION THREAD

**Discussion by sergiofranciscoortiz** [camaraproject/EdgeCloud#197] [Polls]
After an ongoing discussion (https://github.com/camaraproject/EdgeCloud/discussions/196)  on trying to harmonize reference to the same concept to refer to the item returned to either end users or application providers with the information of  where edge applications are deployed, as a way to move forward several options are proposed in this discussion for voting.

With the result of the votation simple_edge_discovery (references to mec-platform) and Edge_LCM APIs ( references to edgecloudnodes) should be updated, some changes in Traffic Influence documentation may apply as well.

Some inputs considered during discussions:

- [Linux Foundation Open Edge glossary](https://stateoftheedge.com/project/glossary/)
- [Current EdgeGroup Glosary](https://github.com/camaraproject/EdgeCloud/blob/main/documentation/SupportingDocuments/edge_terminology.md), it might need to be updated
- [CAMARA commonalities guidelines to reduce telco-specific terminology/acronyms in developer-facing APIs](https://github.com/camaraproject/Commonalities/blob/main/documentation/API-design-guidelines.md#25-reduce-telco-specific-terminology-in-api-definitions)

Apparently only 8 options are available in a poll discussion, if you prefer another option please add that as a comment.

**Comment by sergiofranciscoortiz** [comment]
Please when voting add your name and company that you represent in comments, as indicated in general CAMARA meetings there should be just one vote per company.

**Comment by javierlozallu** [comment]
Javier Loza from Telefónica. 
My vote was for EdgeCloudZone option.

**Comment by Kevsy** [comment]
Kevin Smith Vodafone, voted for EdgeCloudZone

**Comment by gainsley** [comment]
EdgeXR is also happy with EdgeCloudZone.

**Reply by sergiofranciscoortiz** [reply]
Thanks @gainsley, I think that your vote is not accounted in the Poll, could you double check if you actually selected your desired option and clicked on Vote ( if you still has the option to click on it it means that you hadn´t yet voted).

**Reply by surajjj** [reply]
Surajj Jaggernath, Vodacom, voted for EdgeCloudZone

**Reply by gainsley** [reply]
Thanks, you're right, I just added a comment. I have now voted on the poll.

**Comment by nicolacdnll** [comment]
Nicola Cadenelli from Nearby Computing, voted for EdgeCloudZone.

**Comment by sergiofranciscoortiz** [comment]
I'm closing this discussion, the poll was opened during 10 days, in the end all those who vote chose the same option EdgeCloudZone.
